tests/e2e-evm/eslint.config.mjs

```diff
@@ -0,0 +1,30 @@
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  //...tseslint.configs.recommendedTypeChecked,
```

Comment:
Should we keep this here?

Reply:
It doesn't really matter either way imo -- it's just a reminder of the default config, but does not have to be kept.